app/views/spam2/_queue.html.erb

```diff
@@ -27,7 +27,11 @@ $(document).ready(function () {
 </script>
 
 <div class="py-3 pl-3 w-100">
-  <% if @tags_followed.length == 20 %> <a class="fa fa-plus-circle text-info" href="/subscriptions"> </a> <% end %>
+<% if @tags_followed.length == 0 %> 
```

Comment:
@dms-yondy  :tada: :100: **Awesome**!! Can you please format it a bit and add spaces wherever needed ??
Thanks!!

Reply:
@keshavsethi Thanks! I improved the formatting, does it look okay now?